src/lava/lib/optimization/solvers/generic/solver.py

```diff
@@ -405,6 +431,7 @@ def _get_run_config(
         )
 
         if backend in CPUS:
+            # just add CPU processes here in the dictionary for QP

```

Comment:
Please delete the comment above, it seems to be a leftover from the development syncs.